SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -114,7 +114,8 @@ class SPATIALGDK_API USpatialClassInfoManager : public UObject
 	ESchemaComponentType GetCategoryByComponentId(Worker_ComponentId ComponentId);
 	const TArray<Schema_FieldId>& GetFieldIdsByComponentId(Worker_ComponentId ComponentId);
 
-	Worker_ComponentId GetComponentIdForClass(const UClass& Class) const;
+	Worker_ComponentId GetComponentIdForActorClass(const UClass& Class) const;
```

Comment:
Hmm, do we want to make this take a `const TSubclassOf<AActor>`?

Reply:
though Unreal likes to throw around random UClass pointers, so may make usage more difficult

Reply:
TSubclassOf is implicitly constructible from UClass pointer, so this shouldn't change anything from the caller's POV (while defying the point of using it in the first place, as there aren't a lot of runtime checks :/)